tests/integration/functional/model/test_model_version.py

```diff
@@ -62,6 +64,12 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
             pass
 
 
+@step
+def step_metadata_logging_functional():
```

Comment:
Sorry for the nit, but makes it a bit easier to navigate sometimes... Could you add a docstring explaining what this tests?

Reply:
Fixed